v3-sdk/collecting-fees/src/example/Example.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import './Example.css'
+import { CollectOptions, NonfungiblePositionManager } from '@uniswap/v3-sdk'
+import { CurrencyAmount } from '@uniswap/sdk-core'
+import { Environment, CurrentConfig } from '../config'
+import { getCurrencyBalance } from '../libs/wallet'
+import {
+  connectBrowserExtensionWallet,
+  getProvider,
+  TransactionState,
+  sendTransaction,
+  getWalletAddress,
+} from '../libs/providers'
+import {
+  MAX_FEE_PER_GAS,
+  MAX_PRIORITY_FEE_PER_GAS,
+  NONFUNGIBLE_POSITION_MANAGER_CONTRACT_ADDRESS,
+  DAI_TOKEN,
+  USDC_TOKEN,
+} from '../libs/constants'
+import { getPositionIds, mintPosition } from '../libs/liquidity'
+import { fromReadableAmount } from '../libs/conversion'
+
+async function collectFees(positionId: number): Promise<TransactionState> {
```

Comment:
Should this be in liquidity.ts as well?

Reply:
Since this the main function of the example, it feels more complete to keep it in the example

Reply:
Good leaving for now but we should standardize this - I think the goal should be to not store core logic in React related files just to clearly delineate that the React files are for UI of the example and the `libs` can be used in any Typescript application with near copy/paste ability